commit efd89e16f648067e79d43207fc5bc5181045406c
Author: Takashi Iwai <tiwai@suse.de>
Date:   Tue Feb 28 17:27:57 2017 +0100

    ALSA: hda - Add subwoofer support for Dell Inspiron 17 7000 Gaming
    
    commit 493de342748cc6f52938096f5480cf291da58a0b upstream.
    
    Dell Inspiron 17 7000 Gaming laptop needs a similar quirk like
    Inspiron 7599 to support its subwoofer speaker.
    
    Bugzilla: https://bugzilla.kernel.org/show_bug.cgi?id=194191
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1237_efd89e16f648067e79d43207fc5bc5181045406c.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1237_efd89e16f648067e79d43207fc5bc5181045406c.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
